extensions/ql-vscode/package.json

```diff
@@ -1114,6 +1118,11 @@
           "group": "queriesPanel@1",
           "when": "view == codeQLQueries && viewItem == queryFile && codeQL.currentDatabaseItem"
         },
+        {
+          "command": "codeQLQueries.runVariantAnalysisContextMenu",
+          "group": "queriesPanel@1",
+          "when": "view == codeQLQueries && viewItem == queryFile"
```

Comment:
Good question! Though we don't really have a way of checking that a repository is selected (there's no variant analysis version of `codeQL.currentDatabaseItem`) 🤔 

At the moment, you get the same error message that you'd otherwise get when running a variant analysis without selection: 
![image](https://github.com/github/vscode-codeql/assets/42641846/e98001f3-6665-4a02-921a-9b0c67a915ec)